db/schema.rb

```diff
@@ -51,5 +51,13 @@
   add_index "facts_bookings", ["dimensions_date_id"], name: "index_facts_bookings_on_dimensions_date_id", using: :btree
   add_index "facts_bookings", ["reference_number"], name: "index_facts_bookings_on_reference_number", unique: true, using: :btree
 
+  create_table "imports", force: :cascade do |t|
+    t.string   "importer"
+    t.integer  "inserted_records"
+    t.jsonb    "log"
```

Comment:
Log data doesn't really belong in the database. If there are specific audit items or flags we want to record against a fact or dimension then I think we should establish what those are add them to our dimensional model.